tests/sentry/api/test_id_or_slug_path_parameters.py

```diff
@@ -236,12 +374,17 @@ def test_if_endpoints_work_with_id_or_slug(self):
         all_url_patterns = extract_all_url_patterns(root_resolver.url_patterns)
 
         for pattern, callback in all_url_patterns:
-            slug_path_params = extract_slug_path_params(pattern)
-            if slug_path_params:
-                if slug_path_params == ["doc_integration_slug"] or slug_path_params == [
-                    "sentry_app_slug"
-                ]:
-                    non_slug_params = extract_non_slug_path_params(pattern)
-                    self.convert_args_setup_registry[callback.convert_args](
-                        callback, slug_path_params, non_slug_params
-                    )
+            if hasattr(callback, "convert_args"):
+                slug_path_params = extract_slug_path_params(pattern)
+                if slug_path_params:
+                    if slug_path_params == ["doc_integration_slug"] or slug_path_params == [
+                        "sentry_app_slug"
+                    ]:
+                        self.convert_args_setup_registry[callback.convert_args](
+                            callback, slug_path_params
+                        )
+                    elif slug_path_params == ["organization_slug"]:
+                        if self.convert_args_setup_registry.get(callback.convert_args):
+                            self.convert_args_setup_registry[callback.convert_args](
+                                callback, slug_path_params
+                            )
```

Comment:
can use [walrus operator](https://www.codeconquest.com/blog/python-walrus-operator-uses-and-controversy/#htoc-python-walrus-operator-with-if-statement) to combine variable setting and get
```suggestion
                        if convert_args := self.convert_args_setup_registry.get(callback.convert_args):
                            convert_args(
                                callback, slug_path_params
                            )
```